Browser_IDE/editorMain.js

```diff
@@ -418,7 +418,13 @@ function SwitchToTab(editor){
     }
 
     for (let i = 0; i < editors.length; i ++) {
+        // Before refreshing the editor, get the scroll position of the editor window
+        let display = currentEditor.display;
```

Comment:
`display` is a member of `currentEditor.editor`, not `currentEditor` itself. `display` will always be undefined with this code.